nomad/structs/testing.go

```diff
@@ -90,7 +90,10 @@ func MockNode() *Node {
 		Status:                NodeStatusReady,
 		SchedulingEligibility: NodeSchedulingEligible,
 	}
-	node.ComputeClass()
+	err := node.ComputeClass()
+	if err != nil {
+		panic(fmt.Sprintf("failed to compute node class: %v", err))
```

Comment:
It's unfortunate that we don't have these functions in a package where they can never be called from production code. But a quick look for all callers shows we're safe.